README.md

```diff
@@ -52,6 +52,45 @@ $ ./ustreamer --help
 AUR has a package for Arch Linux: https://aur.archlinux.org/packages/ustreamer. It should compile automatically with OpenMAX IL on Raspberry Pi, if the corresponding headers are present in ```/opt/vc/include```.  
 FreeBSD port: https://www.freshports.org/multimedia/ustreamer.
 
+## Building (Docker on Raspbian on Raspberry Pi 4)
+
+Build ustreamer:
+
+```bash
+docker build --file pkg/docker/Dockerfile --tag ustreamer .
+```
+
+Set the hdmi-to-csi bridge EDID:
```

Comment:
I take this is a specific step required for the tc358743-edid - I wouldn't include it in this part of the documentation which should focus on docker